src/models/dissolutionSession.ts

```diff
@@ -0,0 +1,3 @@
+export interface DissolutionSession {
```

Comment:
Hmmm, should we make this default as we don;t expect to need to export other things from this file

Reply:
I agree, I can't see anything else being added to this